config/hashing.php

```diff
@@ -30,21 +30,4 @@
 	'bcrypt' => [
 		'rounds' => env('BCRYPT_ROUNDS', 10),
 	],
-
-	/*
-	|--------------------------------------------------------------------------
-	| Argon Options
-	|--------------------------------------------------------------------------
-	|
-	| Here you may specify the configuration options that should be used when
-	| passwords are hashed using the Argon algorithm. These will allow you
-	| to control the amount of time it takes to hash the given password.
-	|
-	*/
-
-	'argon' => [
-		'memory' => 1024,
-		'threads' => 2,
-		'time' => 2,
-	],
```

Comment:
At the moment, Lychee does not use Argon as a hashing algorithms anywhere. Hence, these options are probably not used currently. I agree on that, even though I am not that sure about any Laravel internal code which might even use Argon.

But what happens if someone starts to use Argon? Will it fail? (I would even call this a good case) Or will it silently fall back to insecure defaults if no settings are found? (This would be the an extremely bad case.) Have you checked that?

Personally, I feel unsure about the (possibly bad) consequences of removing these settings even though I am generally in favor of nuking everything which is unnecessary.